src/routes/Review/index.jsx

```diff
@@ -237,9 +245,9 @@ const Review = ({
                 <PageUl>
                   <li>
                     <strong>
-                      Your Dashboard is your go-to hub for managing your work.
+                      Your Dashboard is your go-to-hub for managing your work.
```

Comment:
Calling something your "go-to" means it's the preferred way of doing that task. A "go-to-hub" doesn't make sense in this context, so I would revert the addition of the extra `-` and leave it as "go-to hub".

Reply:
Done